web/src/engine/main/src/keymanEngine.ts

```diff
@@ -310,12 +310,22 @@ export default class KeymanEngine<
   }
 
   public getDebugInfo(): Record<string, any> {
```

Comment:
When we log errors and events related to KeymanWeb via Sentry:

https://github.com/keymanapp/keyman/blob/4a53cf60c9a6844931bc4f94270756c66655ce8d/common/web/sentry-manager/src/index.ts#L101-L107